### ISSUE TITLE
Breadcrumbs: A11y Issue - The role of current page text should be 'text'

### ISSUE THREAD

**Issue by sleric1024** [SAP/ui5-webcomponents#5717]
**Describe the bug**
When reading "current page" (**Notebooks**) its announced as a "link" while its not a link but just plain text

**Isolated Example**
https://codesandbox.io/s/competent-smoke-mqee3x?file=/src/App.js

**To Reproduce**
1. Go to https://mqee3x.csb.app/
2. Inspect with Chrome developer tool
3. Check the text **Notebooks**
4. The role is marked as 'link' but it's just a plain text

**Expected behavior**
The role should be text

**Screenshots**
![image](https://user-images.githubusercontent.com/5724592/186556713-ae68b0ac-c9b0-4077-873f-82cf7e9c7df4.png)

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 1.6.0
`@ui5/webcomponents-react` version: 0.27.0
Operating System: MacOS
Browser: Chrome

**Additional context**
I am from SAP Successfactors, and this issue is reported from SF a11y expert.

**Comment by PetyaMarkovaBogdanova** [comment]
Hi colleagues from @SAP/ui5-webcomponents-topic-p ,
can you, please, check if the current location element intentionally has the role of a link?
Thank you!

Best regards,
Petya Markova.
(UI5 WC Dispatcher)

**Comment by nnaydenow** [comment]
Hi @sleric1024 ,

ui5-breadcrumbs is navigation with set of links. By W3C best practices last link should be marked as link even if it is only text. The best practices are analysed by our accessibility experts and seems that the component is implemented by their spec.

@dobrinyonkov 

Regards,
Nayden

**Comment by anuvenkatesh1** [comment]
@nnaydenow 
Please see W3C example of breadcrumbs. https://www.w3.org/WAI/ARIA/apg/example-index/breadcrumb/index.html
![image](https://user-images.githubusercontent.com/32471459/190042942-2cc56c23-7a9f-4cb1-93aa-936db3e90b5b.png)
Their "current page" is a "link" which links back to same page and not a "plain text"
However your implementation differs.

**Comment by dobrinyonkov** [comment]
Specification update in progress.
It involves work from our IXD team. An item is created on their board and work is planned to start with the next development takt.

**Comment by nikolay-kolarov** [comment]
I am setting this one to blocked until we have the UX design

**Comment by dobrinyonkov** [comment]
Internal Reference: FIORIDESIGN-7374

**Comment by dobrinyonkov** [comment]
Closed by https://github.com/SAP/ui5-webcomponents/pull/8206
